Database/Entities/Creature.cs

```diff
@@ -1,24 +1,82 @@
 ﻿namespace Database.Entities;
 
+/// <summary>
+/// Represents an in-game creature and it's data.
+/// Creatures belong to a specific faction and are categorized by tiers, with Tier 1 being the weakest 
+/// and Tier 7 being the strongest. <para/>
+/// 
+/// The <c>Creature</c> class provides properties to store and update the unique identifier, faction id,
+/// tier, base cost, upgrade cost, growth and image data for it's base, upgraded and alternate forms. <para/>
+/// 
+/// The binary image data stored in <c>ImageBytesBase</c>, <c>ImageBytesUpg</c> and <c>ImageBytesUpgAlt</c> 
+/// can be converted to a visual representation using the <c>System.Drawing.Image</c> class. <br/>
+/// Note that <c>System.Drawing</c> might require additional dependencies on some platforms. <para/>
+/// 
+/// <b>Additionally</b>, it includes a <c>Faction</c> entity to which it belongs to.
+/// </summary>
 public partial class Creature
 {
+    /// <summary>
+    /// Gets or sets the unique identifier.
+    /// This is a 32-bit signed integer.
+    /// </summary>
```

Comment:
"Gets or sets the unique identifier" is redundant, since the getter and setter will appear on hover. As will the data type.
However what "unique identifier" it is talking about is not clear. As is why can it be set when the primary key should not be able to be altered.

Better alternatives:

> Unique identifier of this record in the database.

> Value of this database record's primary key.